src/Deprecations.jl

```diff
@@ -19,3 +19,8 @@
 @deprecate MatrixSpace(R::Ring, n::Int, m::Int, cached::Bool) MatrixSpace(R, n, m, cached = cached)
 
 @deprecate MatrixAlgebra(R::Ring, n::Int, cached::Bool) MatrixAlgebra(R, n, cached = cached)
+
+function (G::Generic.SymmetricGroup)()
```

Comment:
Why not @deprecate? This version does not print the backtrace, so I don't know where it is called :(

Reply:
no deprecations are showed in and after julia-1.5; use `julia --depwarn=yes` to see them;

moreover `@deprecate` macro does not parse `(G::Generic.SymmetricGroup)()`;

Reply:
I am running it with `depwarn=yes`:
```julia
julia> SymmetricGroup(3)()                                                                                
┌ Warning: (::SymmetricGroup)() to get the group identity is deprecated, use one(::SymmetricGroup) instead
│   caller = ip:0x0                                                                                       
└ @ Core :-1                                                                                              
()                                                                                                        
```
No backtrace.

Reply:
I am not talking about seeing it or not, but about the backtrace.

Reply:
Annoying that the macro does not work :-(